src/stores/store-helper.js

```diff
@@ -0,0 +1,30 @@
+export const getTagsQuery = searchString => [
+  { $unwind: '$tags' },
+  {
+    $match:
+    {
+      $and: [
+        { tags: { $regex: `.*${searchString}.*`, $options: 'i' } },
+        { slug: { $ne: null } } // Is this still needed?
```

Comment:
I am wondering if this condition is (still) needed actually

Reply:
Let's remove it, it doesn't do anything anyways

Reply:
👍 Removed and reduced tag searching to documents, like discussed